internal/assets/manifests/istio-discovery/values.yaml.tpl

```diff
@@ -107,11 +107,29 @@ pilot:
 {{ $x | indent 2 }}
 {{- end }}
 
-{{- if .GetSpec.GetHttpProxyEnvs }}
```

Comment:
Maybe I'm missing sth but just to double check is this deliberately deleted?

Reply:
What about this comment?

Reply:
before this feature, this field was only used in case proxyenvs is set (see next 2 lines). now these values will be set either way.

Reply:
Why exactly will "these values will be set either way"?

Isn't there a matching `end` section that should be deleted in this case as well?

Reply:
the matching end is not detected in the diff, see line 114

Reply:
Ah, I see, thanks!